dev/reference/performance/statement-summary.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Statement Summary Table
+category: reference
+---
+
+# Statement Summary Table
+
+## 前言
+
+我们在排查线上 TiDB 问题时，可能有下面这样的需求：
+
+- SQL 延迟比较大，是不是服务端的问题？
+- 哪类 SQL 的总耗时最高？
+
+遇到这类需要定位 SQL 的问题，我们会首先想到两种方式来排查：
+
+- 打开 general log，但是打印 general log 对 Server 的性能会有影响。
+- `admin show slow` 显示慢查询日志，但是有问题的 SQL 可能没有被归为慢 SQL。
+
```

Comment:
Suggest we remove this kind of info, which is redundant to users. In this section, we only need to introduce what is statement summary table and what this document is about. @djshow832

Reply:
This way we also have a more brief introduction section.